Sources/TuistGraph/Models/Project.swift

```diff
@@ -75,20 +61,22 @@ public struct Project: Hashable, Equatable, CustomStringConvertible, CustomDebug
     ///   - targets: The project targets
     ///   - additionalFiles: The additional files to include in the project
     ///                      *(Those won't be included in any build phases)*
-    public init(path: AbsolutePath,
-                sourceRootPath: AbsolutePath,
-                xcodeProjPath: AbsolutePath,
-                name: String,
-                organizationName: String?,
-                developmentRegion: String?,
-                settings: Settings,
-                filesGroup: ProjectGroup,
-                targets: [Target],
-                packages: [Package],
-                schemes: [Scheme],
-                ideTemplateMacros: IDETemplateMacros?,
-                additionalFiles: [FileElement])
-    {
+    public init(
```

Comment:
Maybe here also we are missing the `resourceSynthesizers ` documentation in init

Reply:
I think this one is fine without a comment as it's also in `TuistGraph`